source/assets/js/components/fef-selectable.js

```diff
@@ -120,7 +121,7 @@ export class SrfSelectableCollection {
      */
     showSelectionElement(event) {
         let $collection = this.$sourceCollections.filter(`.${SELECTED_COLLECTION_CLASS}`).first(),
-            shouldFocus = event.screenX === 0 && event.screenY === 0;
+            shouldFocus = FefTouchDetection.eventIsMouseclick(event);
```

Comment:
War das nicht grad andersrum?

Reply:
momol

Reply:
äbe drum macht so ein sprechender Methodenname schon noch Sinn